### ISSUE TITLE
Azerty keyboard layout not working inside the editor on OS X

### ISSUE THREAD

**Issue by NindroidX** [godotengine/godot#808]
It seems that for some reason the Godot editor does not recognise my Azerty keyboard layout. It thinks that I am using a Qwerty keyboard.

For example:
When I want to undo an operation I need to press "Meta + W" instead of "Meta + Z". If you compare Qwerty and Azerty, you see that the W and Q keys are reversed. Other shortcuts show the same behaviour: "Meta + A" and "Meta + Q" are flipped as well.

The strange thing is, they keys work as expected inside the script editor..

**Comment by MangelMaxime** [comment]
I think this issue can be closed. Not the shortcut are no longer flipped under OS X azerty.

**Comment by vnen** [comment]
I believe it's the same as #1106 and by so was fixed in #1151.

**Comment by green-coder** [comment]
I am using the standard QWERTY keyboard of a MacBookAir which I converted into the "Programmer Dvorak" via physical key relocation (I moved them, physically) and via software (I am using the Programmer Dvorak layout in the Mac OSX settings).

The editor of Godot 3.0.2 still think that I am using the QWERTY layout when I type the shortcuts (e.g. Ctrl+S, Ctrl+Z, etc...). The text I type when I rename the nodes is correct.

Should I open a new issue of ask to reopen this one?
